cmd/podRestartsExternalCheck/main.go

```diff
@@ -0,0 +1,263 @@
+// Package podRestarts implements a checking tool for pods that are
+// restarting too much.
+
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"strconv"
+	"sync"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	checkclient "github.com/Comcast/kuberhealthy/pkg/checks/external/checkClient"
+	"github.com/Comcast/kuberhealthy/pkg/kubeClient"
+)
+
+const maxFailuresAllowed = 5
```

Comment:
not sure if we want to make this configurable and set it as an optional env var with a default set to 5?

Reply:
i would definitely like to see this configurable.  Here is an issue for it: #225